independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartCatalog.java

```diff
@@ -115,6 +115,8 @@ public static QuarkusCodestartCatalog fromExtensionsCatalog(ExtensionCatalog cat
 
     @Override
     protected Collection<Codestart> select(QuarkusCodestartProjectInput projectInput) {
+        // TODO: Add logic for group id - Like setting by default the parent groupid/version when these are empty
```

Comment:
I am not sure, but I think I would do that before (maybe in `CreateProjectCommandHandler`) to keep the codestart generation part as dumb as possible wdyt?